.ci/scripts/prepare_tool.sh

```diff
@@ -4,7 +4,7 @@
 # found in the LICENSE file.
 
 # To set FETCH_HEAD for "git merge-base" to work
-git fetch origin master
+git fetch origin main
```

Comment:
I believe this is operating on the flutter/flutter repo

Reply:
It is running on flutter/packages. Seems it's passing: https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8812341372307443777/+/u/Run_package_tests/prepare_tool/stdout

Reply:
No it is on flutter/package, this is used for `git merge-base` to get the diff work. 

We also need to do the same for plugins.